util/helpers.py

```diff
@@ -0,0 +1,9 @@
+
+def keep_only_digits(txt):
+    return ''.join(filter(lambda c: c.isdigit(), txt))
```

Comment:
nit: `str.isdigit` instead of `lambda c: c.isdigit()`.

Reply:
Thought the same - didn't work as `self` is a parameter that also has to be bound.

Reply:
> In [1]: ''.join(filter(str.isdigit, 'aeiou1234'))
> Out[1]: '1234'

Reply:
`self` will be bound when called by filter, it's just the first parameter.

In [4]: str.isdigit('1')
Out[4]: True

In [5]: str.isdigit('a')
Out[5]: False

Reply:
Ah - got it.